site/content/contribute/more-info/desktop/developer-setup/macos.md

```diff
@@ -3,19 +3,7 @@
 3. Install dependencies
 
     ```sh
-    brew install node@16 git python3
+    brew install nvm git python3
```

Comment:
We might want to recommend installing nvm another way since managing nvm via Homebrew is not officially supported by nvm. I received a warning when I initially installed it which prompted me to uninstall it and install it the officially supported way.

Note about not officially supported: https://formulae.brew.sh/formula/nvm

Officially supported installation: https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating

Reply:
I think I'm fine with this. We are using `brew` to install the other dependencies as well and unless we expect or see some issues with this installation method I think I'd rather make it as easy as possible to install. It's always worked for me.